README.md

```diff
@@ -15,53 +15,30 @@ Build your own custom MLOps orchestration workflows from composable automation r
-
-## FuseML Principles
-
-* *Flexibility* - create and manage dynamic MLOps workflows connecting different AI/ML tools across multiple infrastructure domains
-
-* *Extensibility* - leverage FuseML's set of abstractions and extension mechanisms to add support for your favorite AI/ML tools
-
-* *Composability* - build complex MLOps workflows for your projects out of composable building blocks implementing a wide range of machine learning functions     
-
-* *Collaboration* - use MLOps automation and tool integration recipes created in collaboration by all AI/ML team roles - Data Scientists, Data Engineers, and DevOps Engineers
-
-* *GitOps for Machine Learning* - extend what traditional GitOps does with code to all other types of artifacts specific to machine learning - models and datasets, as well as other byproducts of a DevOps workflow (e.g. packages, container images) to provide features such a end-to-end versioning, tracking and reproducibility
-
-## Inception and Direction
-
-FuseML originated as a fork of our sister open source project [Carrier](https://github.com/SUSE/carrier), a lightweight open source PaaS built on top of Kubernetes, then has been gradually transformed and infused with the MLOps concepts that make it the AI/ML orchestration tool that it is today.
-
-The FuseML team is still in the process of defining the project strategy, but we can say with some certainty that the project will develop in the following main directions aligned with the stated principles:
+The project is under heavy development following the main directions:
 1. adding features and enhancements to improve flexibility and extensibility
-2. add support for more community shared AI/ML tools
-3. create more composable automation blocks adapted to the existing as well as new AI/ML tools
+2. adding support for more community shared AI/ML tools
+3. creating more composable automation blocks adapted to the existing as well as new AI/ML tools
+
+Take a look at [our Project Board](https://github.com/orgs/fuseml/projects/1) to see what we're working on and what's in store for the next release.
 
 ## Basic Workflow
 
-The basic FuseML workflow can be described as a GitOps type of workflow that starts with your ML code and automatically runs all the steps necessary to build and serve your machine learning model. FuseML's job begins when your machine learning code is ready for execution.
+The basic FuseML workflow can be described as an MLOps type of workflow that starts with your ML code and automatically runs all the steps necessary to build and serve your machine learning model. FuseML's job begins when your machine learning code is ready for execution.
 
 1. install the FuseML Ochestrator in a kubernetes cluster of your choice (see [Installation Instructions](# Install))
```

Comment:
Maybe just `install FuseML in a kubernetes...`

Also the link to the installation instructions is missing.

Reply:
thanks, would've missed that